website/content/en/preview/concepts/node-templates.md

```diff
@@ -421,6 +438,10 @@ spec:
 
 For more examples on configuring these fields for different AMI families, see the [examples here](https://github.com/aws/karpenter/blob/main/examples/provisioner/launchtemplates).
 
+{{% alert title="Windows Support Notice" color="warning" %}}
+Specifying `spec.userData` is not currently supported for Windows nodes, but support is expected in the future.
```

Comment:
https://github.com/aws/karpenter/issues/4081

Reply:
Great. @tzifudzi can you include this in the website here, and likewise for the other issues and their spots? That way, users can track it, when they see this rather than having to go find the issue. Likewise, when it gets implemented, this should be modified and the link can be removed anyway.

Reply:
Added the link into the docs here